README.md

```diff
@@ -313,6 +313,34 @@ will hopefully enable both quit-if-one-screen and mouse-wheel scrolling.
 
 If scrolling still doesn't work for you, you can try to pass the `-S` option in addition.
 
+### Configuration file
+
+`bat` can use a config file for options to cut down on command line flags, that you want to save for each run. 
+
+#### Location 
+
+The default location is operation system/install location dependent (macOS: `Users/<your_user>/Library/Preferences/bat/config`). To get the path of the current location use `bat --config-file` to print out the location of the current config file path.  
```

Comment:
It feels a bit weird that we only mention the path for `macOS`. I would remove the part in parenthesis and just say "To get the path for your operating system, use ...".